internal/util/update/update_test.go

```diff
@@ -815,6 +815,125 @@ func TestCommand_Run_failInvalidRef(t *testing.T) {
 	}
 }
 
+func TestCommand_Run_manualChange(t *testing.T) {
```

Comment:
What if user changes Kptfile and also triggers cli command with different version ? Cli command takes precedence ?

Reply:
Yeah, the information provided through flags will be written to the Kptfile and then we go through the update process same as if it was updated manually.